tikv-client/src/main/java/com/pingcap/tikv/jdbc/TiDBJDBCClient.java

```diff
@@ -157,8 +173,17 @@ public void updateTableStatistics(long startTS, long tableId, long delta, long c
     }
   }
 
+  public void updateTableStatisticsWithRetry(long startTS, long tableId, long delta, long count)
+      throws SQLException {
+    executeWithUpdateConnRetry(
+        () -> {
+          updateTableStatistics(startTS, tableId, delta, count);
+          return null;
```

Comment:
why return null

Reply:
Maybe there is a better way to solve it, but I have no idea now.
"why return null" here is due to `void` can not be used for generics, so `return null` is used instead of Void.